include/pinocchio/bindings/python/spatial/inertia.hpp

```diff
@@ -295,6 +312,251 @@ namespace pinocchio
 
     }; // struct InertiaPythonVisitor
 
+    template<typename PseudoInertia>
+    struct PseudoInertiaPythonVisitor
+    : public boost::python::def_visitor<PseudoInertiaPythonVisitor<PseudoInertia>>
+    {
+      enum
+      {
+        Options = PseudoInertia::Options
+      };
+      typedef typename PseudoInertia::Scalar Scalar;
+      typedef typename PseudoInertia::Vector3 Vector3;
+      typedef typename PseudoInertia::Matrix3 Matrix3;
+      typedef typename PseudoInertia::Vector10 Vector10;
+      typedef typename PseudoInertia::Matrix4 Matrix4;
+      typedef Eigen::Matrix<Scalar, Eigen::Dynamic, 1, Options> VectorXs;
+
+    public:
+      template<class PyClass>
+      void visit(PyClass & cl) const
+      {
+        cl.def(bp::init<const Scalar &, const Vector3 &, const Matrix3 &>(
+                 (bp::arg("self"), bp::arg("mass"), bp::arg("h"), bp::arg("sigma")),
+                 "Initialize from mass, vector part of the pseudo inertia and matrix part of the "
+                 "pseudo inertia."))
+          .def(bp::init<const PseudoInertia &>(
+            (bp::arg("self"), bp::arg("clone")), "Copy constructor"))
+          .add_property(
+            "mass", &PseudoInertiaPythonVisitor::getMass, &PseudoInertiaPythonVisitor::setMass,
+            "Mass of the Pseudo Inertia.")
+          .add_property(
+            "h", &PseudoInertiaPythonVisitor::getH, &PseudoInertiaPythonVisitor::setH,
+            "Vector part of the Pseudo Inertia.")
+          .add_property(
+            "sigma", &PseudoInertiaPythonVisitor::getSigma, &PseudoInertiaPythonVisitor::setSigma,
+            "Matrix part of the Pseudo Inertia.")
+
+          .def(
+            "toMatrix", &PseudoInertia::toMatrix, bp::arg("self"),
+            "Returns the pseudo inertia as a 4x4 matrix.")
+          .def(
+            "toDynamicParameters", &PseudoInertiaPythonVisitor::toDynamicParameters_proxy,
+            bp::arg("self"), "Returns the dynamic parameters representation.")
+          .def(
+            "FromDynamicParameters", &PseudoInertia::template FromDynamicParameters<VectorXs>,
```

Comment:
This code can crash.

There is no runtime check to verify that the input matrix is Vector10.
InertiaTpl suffer from the same issue and I was able to crash the interpreter with the following code

```python
pin.Inertia.FromDynamicParameters(np.array([]))
```